src/services/githubAPI.ts

```
@@ -1,82 +1,425 @@
+    }
+
+    const { owner, repo, type, identifier } = parsed;
+
+    switch (type) {
+      case 'commit':
+        if (!identifier) throw new Error('Commit SHA not found in URL');
+        return this.getCommitDiff(owner, repo, identifier);
+        
+      case 'compare':
+        if (!identifier) throw new Error('Comparison range not found in URL');
+        const [base, head] = identifier.split('...');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)